scripts/snip.py

```diff
@@ -158,6 +167,9 @@
 
 with open(os.path.join(snipdir, snipfile), 'w', encoding='utf-8') as f:
     f.write(HEADER % markdown.split("content/en/")[1] if "content/en/" in markdown else markdown)
+    for bp in boilerplates:
+        source_line = f"source 'content/en/boilerplates/snips/{bp}.md_snips.sh'\n"
```

Comment:
I think the file name can be a little simpler (remove the `.md_snips` part. Also, use double quotes for the file might be more consistent with the recommended pattern for adding sources manually.
```suggestion
        source_line = f"source \"content/en/boilerplates/snips/{bp}.sh\"\n"
```

Reply:
Done.